operations-stdlib/src/commonMain/kotlin/Size.kt

```diff
@@ -0,0 +1,14 @@
+import operation.StandardLogicOperation
+import utils.asList
+import utils.isSingleNullList
+
+object Size : StandardLogicOperation {
+    override fun evaluateLogic(expression: Any?, data: Any?): Any? {
+        return when (val firstItem = expression.asList.firstOrNull()) {
+            firstItem.isSingleNullList() -> null
```

Comment:
I am not sure if it should be treated as null. We decided to stick to the pure kotlin rules so wdyt about deleting this condition or treat it like empty list/size = 0?

Reply:
so e.g. for which input this operator would return 0?

Reply:
I wrote on PW, but in current way we do things 0 will be returned for listOf(listOf()) added test to verify that